monitoring/managing-alerts.adoc

```diff
@@ -56,6 +56,8 @@ include::modules/monitoring-expiring-silences.adoc[leveloffset=+2]
 // Sending notifications to external systems
 include::modules/monitoring-sending-notifications-to-external-systems.adoc[leveloffset=+1]
 include::modules/monitoring-configuring-alert-receivers.adoc[leveloffset=+2]
+// Creating alert routing for user-defined projects
+include::modules/monitoring-creating-alert-routing-for-user-defined-projects.adoc[leveloffset=+2]
 
 // Applying a custom Alertmanager configuration
```

Comment:
Ah, I see. This procedure has multiple procedures for two different ways to change the alert manager configuration (CLI and OpenShift Container Platform web console). 

I would pick one to focus on. Users don't need to know all the ways. They simply need to know the way that you think you can best document and the way you think best serves them. 

Since all your other procedures are for the CLI, I would keep that way. 

Are you even editing this procedure in this PR? I see here you simply rename it.

Reply:
Another note on this procedure (it's hard to call out specific instances since I don't see the lines here in the PR)

1. additional resources -- preferably, don't link to pages outside RH documentation 
2. there's a lot more subtext to the procedure steps than there are in other procedures. Kind of comes across as inconsistent. Those subtext could use some grammatical changes (minor instances of missing commas). 

Again, is the procedure receiving attention in this PR or is it a separate concern for a separate PR?

Reply:
@jc-berger These are all good points about that chunk of content, but I'm going to mark your comments as resolved since this PR is for adding new content for a different feature.